docs/book/GLOSSARY.md

```diff
@@ -146,6 +166,12 @@ Unless otherwise specified, this refers to horizontal scaling.
 
 The infrastructure that backs a [Machine Resource](#user-content-machine), typically either a cloud instance, virtual machine, or physical host.
 
+# T
+
+## Target cluster
+
+The declared cluster we intend to create and manage using source cluster.
```

Comment:
```suggestion
The declared cluster we intend to create and manage using cluster-api when running `clusterctl create cluster`.

When running `clusterctl alpha phases pivot` this refers to the cluster that will be the new management cluster.
```

Reply:
Thanks for the work on the glossary! However, I thought that we have been using the terms "Management" and "Workload" to refer to clusters? Are these terms not expressive enough?

Reply:
They are not expressive enough to discuss the process of pivoting a management cluster.

Reply:
Good point